torch/csrc/jit/tensorexpr/ir.h

```diff
@@ -35,6 +19,36 @@ enum CompareSelectOperation {
   kNE,
 };
 
+inline int getPrecedence(IRNodeType ty) {
```

Comment:
Please clarify in a comment that whether this is C++ precedence, or Java operator precedence, i.e., whether higher or lower is tighter.

Reply:
It's C++ operator precedence, since we're printing to C++: https://en.cppreference.com/w/cpp/language/operator_precedence

Reply:
Holy magic numbers, Batman!  :-D  Can you put that link in a comment so I don't have to wonder where 5, 6, 11 comes from?

Reply:
Added comments and link